.rubocop.yml

```diff
@@ -1267,4 +1269,485 @@ RSpec/LeakyConstantDeclaration:
 RSpec/DescribedClass:
   Enabled: false
 
+# This is the default configuration file.
```

Comment:
Why add this configuration and not let `rubocop-faker` use its defaults? This has the advantage that everything keeps updated after upgrading gems.

Reply:
Decidim has extended the Faker with some of its own .. config seen there is also addressing the `Decidim::Faker::Localized:`  calls, as defined at https://github.com/decidim/decidim/pull/6764/files#diff-4f894049af3375c2bd4e608f546f8d4a0eed95464efcdea850993200db9fef5cR1722